### ISSUE TITLE
[JED] Unpublished Extensions Error Codes - blank page

### ISSUE THREAD

**Issue by pe7er** [joomla/joomla-websites#1714]
At https://extensions.joomla.org/ at an unpublished extension I clicked "View error codes"
and the resulting page https://extensions.joomla.org/support/knowledgebase/item/unpublished-extensions-error-codes/
resulted in HTTP 200 but was completely empty.

Tested in Google Chrome and Firefox

**Comment by Llewellynvdm** [comment]
I already spend days trying to fix that, but who ever build this, hard coded those links, painful. (since we combed over the whole DB to make sure all links are updated) But thanks for the reminder, we are still at it trying to resolve this since the first time it was pointed out to us. As I remember there is even a forum thread on this right?

**Comment by HLeithner** [comment]
@pe7er is this still valid? @JazParkyn

**Comment by pe7er** [comment]
@HLeithner Yes, https://extensions.joomla.org/support/knowledgebase/item/unpublished-extensions-error-codes/ still triggers a blank page.

**Comment by HLeithner** [comment]
@mfleeson can you have a look at this page?

**Comment by mfleeson** [comment]
The link should be https://extensions.joomla.org/support/knowledgebase/unpublished-extensions-error-codes/ 

This has been changed in the language definitions which is used to present the link to the end user. However I cannot find out how to view an unpublished extension to test if the link is updated on the site.

**Comment by codeling** [comment]
The correct link now actually is 
https://extensions.joomla.org/support/knowledgebase/error-codes/unpublished-extensions-error-codes

> However I cannot find out how to view an unpublished extension to test if the link is updated on the site.

Verifying that the correct link is shown is indeed a bit tricky. But searching for "unpublished site:extensions.joomla.org" lead me to a currently unpublished extension on the second page of search results or so: https://extensions.joomla.org/extension/booklibrary-basic/

So I can confirm that this issue is fixed!

**Comment by particthistle** [comment]
_Community Sites Team Follow up & close_

- A redirect from https://extensions.joomla.org/support/knowledgebase/item/unpublished-extensions-error-codes/ to https://extensions.joomla.org/support/knowledgebase/error-codes/unpublished-extensions-error-codes could be put in place on the Extensions site to completely close the issue (either Joomla Redirects plugin or htaccess)
- if the first link is now not directly linked in the Extensions directory any more, then this page may be the only place it's referenced going forward. If so, issue may be able to be closed?
